lib/opsgenie_tamer/support_rotations_fetcher.rb

```diff
@@ -0,0 +1,71 @@
+module OpsgenieTamer
+  class SupportRotationsFetcher
+    FIRST_LINE_SCHEDULE_ID = "e71d500f-896a-4b28-8b08-3bfe56e1ed76"
+    SECOND_LINE_SCHEDULE_ID = "b8e97704-0e9d-41b5-b27c-9d9027c83943"
+
+    attr_accessor :historical_interval_in_months, :total_interval_in_months
+
+    def initialize(historical_interval_in_months: 2, total_interval_in_months: 12)
+      @historical_interval_in_months = historical_interval_in_months.to_i
+      if @historical_interval_in_months < 1
+        @historical_interval_in_months = 1
+      end
+      if @historical_interval_in_months > 12
+        @historical_interval_in_months = 12
+      end
+      @total_interval_in_months = total_interval_in_months.to_i
+      if @total_interval_in_months < @historical_interval_in_months
+        @total_interval_in_months = @historical_interval_in_months
+      end
+    end
```

Comment:
I was thinking of a way we could simplify this logic, but then I realised we don't actually use this at all. Can we just nail the intervals in to keep the codebase simpler?

Reply:
Hah, yeah, that's a remnant from when I thought we'd be able to offer those as params. /o\ I'm ok with making them constants, so they're not fully "magic numbers".

Reply:
Yep, makes sense 👍